test/instrumentation/run.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/sh
+version=`node -v`
+versionMinusV=`echo $version | cut -c 2-`
+nodeRoot="node-$versionMinusV"
+
+if [ ! -d $nodeRoot ]; then
+  url="https://codeload.github.com/nodejs/node/tar.gz/$version"
```

Comment:
This is true, they're not necessary, but I personally prefer them as well for readability when working with variables.